src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,44 @@
+package mate.academy.service
+
+import mate.academy.figures.Circle
+import mate.academy.figures.Figure
+import mate.academy.figures.IsoscelesTrapezoid
+import mate.academy.figures.Rectangle
+import mate.academy.figures.Square
+import mate.academy.figures.RightTriangle
+import mate.academy.model.Color
+import kotlin.random.Random
+
+const val MIN = 1.0
+const val MAX = 10.0
+const val ONE = 1
+const val TWO = 2
+const val THREE = 3
+const val FOUR = 4
+const val FIVE = 5
+const val DEFAULTRADIUS = 10.0
+
+class FigureSupplier {
+    fun getRandomFigure(): Figure {
+        val colorSupplier = ColorSupplier()
```

Comment:
We don't need to create an instance of the `ColorSupplier()` each time we call `getRandomFigure()` method. Move it to the class fields